integration/run_test.go

```diff
@@ -0,0 +1,356 @@
+/*
+   Copyright The Soci Snapshotter Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package integration
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"testing"
+	"time"
+
+	shell "github.com/awslabs/soci-snapshotter/util/dockershell"
+	"github.com/awslabs/soci-snapshotter/util/dockershell/compose"
+	"github.com/awslabs/soci-snapshotter/util/testutil"
+)
+
+const (
+	drupalImage = "drupal:latest"
+)
```

Comment:
nit: maybe put the const in the `testutil` pkg and export (`DrupalImage`).

Reply:
Moved this and the three other related constants from pull_test, all to testutil.

Why export?

Reply:
just noticed there are two `testutil`. in the 2nd case export is not needed since they are in the same package :)

https://github.com/awslabs/soci-snapshotter/blob/078ae664ed40b06679fce302f792812c97f7adef/util/testutil/util.go

https://github.com/awslabs/soci-snapshotter/blob/078ae664ed40b06679fce302f792812c97f7adef/integration/testutil.go